### DISCUSSION TITLE
Pull Shark? That's pretty patronizing. I do this for a living.

### DISCUSSION THREAD

**Discussion by blanketcat** [community/community#18899] [Other Feature Feedback, Questions, & Ideas]
I don't want badges, I don't want a social network, I want a place to store my code. This isn't an afterschool club for me. I do this for a living. F---ing "Pull Shark"? Patronizing.

I'm almost 40. I don't need little dopamine hits when I'm working on code, I'm working on code, that's what I do for dopamine.

I don't want the services I'm using to RECOGNIZE AND REACT TO WHAT I'M DOING AT ALL. Stop paying attention. I want the services I'm using to function. I want stupid services that work. Hopefully well. Services are not my friends, they're services. This isn't a playground, it's where I work.

I notice it says beta next to the badge. So now you're pushing me into beta programs without asking me if I want to participate. Bold move. I already don't want to use Github after using all of the code you've collected over the years for the AI code assistant that I would have to pay for even though our work made it possible.

GITHUB. STOP. ALL. OF. THIS.

I don't have a facebook account.
I don't have a snapchat account.
I don't have a tiktok account.
Why are you ruining this?

The git server I host at home doesn't do any of this. I don't want to put it behind DynDNS and deal with setting up the firewall, auth etc. That's all Github is. It takes care of some annoying things for me. If it becomes more annoying than the problems it solves, it ceases to be a viable alternative to just doing it myself. Bitbucket and Gitlab don't do any of this social media marketing nonsense. WHY. ARE. YOU. DOING. THIS? This does not improve anything fundamental to the service.

Look, my work uses github. I don't want to use a different service for work and personal projects if I can possibly help it, I have enough context switching with OS, text editor, and frameworks. Please, I'm soooo close to switching, and I don't want to. I don't want to learn another UI. I don't want to update my .gitconfig username and walk away, BUT I WILL IF I LAND IN ONE MORE BETA.

You looking for a strong reaction. Here it is. NO. This is AWFUL. OPT ME OUT. Pull Shark? I send potential employers and clients here sometimes, do I need to worry that they will think I am a child if I continue to use your platform?

WHY ARE YOU TRYING TO FIX SOMETHING THAT WORKS? STOP.

Sincerely,
Me

**Comment by Nik-08** [comment]
For more info, see: https://github.com/Schweinepriester/github-profile-achievements/

**Comment by MichaelOdermatt** [comment]
I'm not sure if its really an opt out but you can remove them from your profile through your settings. https://github.com/settings/profile

![image](https://user-images.githubusercontent.com/43145047/174485643-4b347a3d-d31d-475f-886f-5971410703b5.png)

**Reply by brentlintner** [reply]
It removes all achievements 😞. I wish I could remove just the news ones. They feel superfluous and "gamey".

I want my badges to mean something (ex: arctic doe contrib, sponsor), and look professional. 😢

**Comment by jlumbroso** [comment]
I do understand your concerns @blanketcat (though it's not clear if you are trolling or mean this seriously, given your profile mentions being a GPT-3 troll 😁).

Like you, I have lots of experience. I have been programming since I was five, and I love code so much, I ended up teaching students how to program for a living. Having been around through the birth of the open source era, with GNU, emacs, vim, Linux, and then SourceForge, Napster, Shareaza, Gnutella, etc., ... I can tell you that GitHub has done more good globally for programming literacy and advocacy of open source than any other company on the planet. GitHub has, at scale, had a tremendous impact on programmers by helping them provide organized work. The nudges to submit a README.md, to have rules of conduct, of contribution, to have easy to use tools to submit issues, to have ever more integrated and smart workflows — all of these things help more and more people find it accessible to write *usable* and *reusable* code, and I think that bringing as many creative minds to software engineering is a good thing for the world.

This gamification may feel patronizing to some, and so, it's great that it can be turned off. But it does have a positive impact, especially in the open source community where many, for several decades, were locked in silly flamewars with no real world impact.

To put it another way, I think GitHub is one of those rare platforms that harnesses the value of a social platform the best. It is one of the few social media platforms that is not performative, where what matters is your work. It is a perfectly reasonable position to think that GitHub is a valuable social network, which has supported the growth of many public goods, in opposition to Snapchat, Facebook, Instagram, TikTok, that are advertising companies in disguise — that benefit from engagement at the expense of quality engagement.

**Comment by CarstenSavage47** [comment]
Your perspective is very interesting, blanketcat! I definitely understand where you're coming from. In some ways, Github feels like it's becoming more and more like Steam in a way with these achievements and all. I can't tell if that's a good or bad thing, but it's interesting.

**Comment by georgeharry11** [comment]
True, not everything has to be gamified. Some of us just want tools that work. I felt the same [confusion around SnapTroid](https://snapspector.com/snaptroid/) at first, but after digging deeper I managed to clear up my thoughts about it.
